src/main/java/com/possible/dhis2int/web/DHISIntegrator.java

```diff
@@ -316,29 +326,29 @@ public FileSystemResource downloadSubmissionLog(HttpServletResponse response) th
 
 	@RequestMapping(path = "/download")
 	public void downloadReport(@RequestParam("name") String name, @RequestParam("year") Integer year,
-			@RequestParam("month") Integer month, @RequestParam("isImam") Boolean isImam, HttpServletResponse response)
+							   @RequestParam("month") Integer month, @RequestParam("isImam") Boolean isImam,@RequestParam("isFamily") Boolean isFamily, HttpServletResponse response)
 			throws JSONException, IOException {
 		ReportDateRange reportDateRange = new DateConverter().getDateRange(year, month);
-		if (isImam != null && isImam) {
```

Comment:
@suruchee , are these conditions for Imam intentionally deleted ? if so whats solution for downloading Imam reports in other ways?

Reply:
It was by mistake and I will add it up.